molo/core/models.py

```diff
@@ -822,7 +823,11 @@ class BannerPage(ImportableMixin, TranslatablePageMixin, Page):
                                      help_text='External link which a banner'
                                      ' will link to. '
                                      'eg https://www.google.co.za/')
-    api_fields = ["banner", "banner_link_page", "external_link"]
+    hide_on_freebasics = models.BooleanField(default=False,
```

Comment:
@codiebeulaine I don't think we need this variable since we take care of it with `is_via_freebasics`, instead we should have a checkbox to indicate that if it's a text banner or a normal banner

Reply:
Agreed. Let's call this something else which is handled differently in GEM for freebasics. e.g banner_type? `hide_on_freebasics` would be very confusing for NurseConnect or any other molo project that doesn't known about FreeBasics